src/language/gdj_language.cpp

```diff
@@ -35,6 +36,13 @@ void GdjLanguage::init() {
 #endif
 }
 
+void GdjLanguage::frame() {
+    if (!GDKotlin::get_instance().user_configuration.disable_gc) {
```

Comment:
wouldn't an unlikely here be beneficial?

Reply:
Not really, it's meant to be called once per frame, it has no performance implications